dojo/finding/views.py

```diff
@@ -2084,6 +2084,7 @@ def promote_to_finding(request, fid):
             new_finding.duplicate = False
             new_finding.mitigated = None
             new_finding.verified = True
+            new_finding.verified_date = timezone.now()
```

Comment:
This is the line that should ensure the date is set at the time of verification.